azure_jumpstart_ag/artifacts/Bootstrap.ps1

```diff
@@ -99,10 +100,12 @@ Resize-Partition -DriveLetter C -Size $(Get-PartitionSupportedSize -DriveLetter
 [System.Environment]::SetEnvironmentVariable('AgConfigPath', "$AgDirectory\AgConfig.psd1", [System.EnvironmentVariableTarget]::Machine)
 Invoke-WebRequest ($templateBaseUrl + "artifacts/AgLogonScript.ps1") -OutFile "$AgDirectory\AgLogonScript.ps1"
 Invoke-WebRequest ($templateBaseUrl + "artifacts/AgConfig.psd1") -OutFile "$AgDirectory\AgConfig.psd1"
+Invoke-WebRequest ($templateBaseUrl + "artifacts/icons/grafana.ico") -OutFile $AgIconsDir\grafana.ico
 
 BITSRequest -Params @{'Uri'='https://aka.ms/wslubuntu'; 'Filename'="$AgToolsDir\Ubuntu.appx" }
 BITSRequest -Params @{'Uri'='https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_x64.msi'; 'Filename'="$AgToolsDir\wsl_update_x64.msi"}
 BITSRequest -Params @{'Uri'='https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe'; 'Filename'="$AgToolsDir\DockerDesktopInstaller.exe"}
+BITSRequest -Params @{'Uri'='https://dl.grafana.com/oss/release/grafana-9.4.7.windows-amd64.msi'; 'Filename'="$AgToolsDir\grafana-9.4.7.windows-amd64.msi"}
```

Comment:
is pinning to this specific version something we can make more dynamic to keep up with newer releases?